imap_processing/lo/l1a/lo_l1a.py

```diff
@@ -231,5 +291,20 @@ def add_dataset_attrs(
                 "events",
             ]
         )
+        # An empty DEPEND_0 is being added to support_data
+        # variables that should only have DEPEND_1
+        # Removing Depend_0 here.
+        # Should look for a fix to this issue
+        del dataset["direct_events"].attrs["DEPEND_0"]
+        del dataset["coincidence_type"].attrs["DEPEND_0"]
+        del dataset["de_time"].attrs["DEPEND_0"]
+        del dataset["mode"].attrs["DEPEND_0"]
+        del dataset["esa_step"].attrs["DEPEND_0"]
+        del dataset["tof0"].attrs["DEPEND_0"]
+        del dataset["tof1"].attrs["DEPEND_0"]
+        del dataset["tof2"].attrs["DEPEND_0"]
+        del dataset["tof3"].attrs["DEPEND_0"]
+        del dataset["pos"].attrs["DEPEND_0"]
+        del dataset["cksm"].attrs["DEPEND_0"]
```

Comment:
loop over the options
```suggestion
        for var in ["direct_events", ...]:
            dataset[var].attrs.pop("DEPEND_0")
```